src/three/TilesRenderer.js

```diff
@@ -600,7 +601,8 @@ export class TilesRenderer extends TilesRendererBase {
 
 		}
 
-		switch ( extension ) {
+		const fileType = readMagicBytes( buffer ) || extension;
```

Comment:
In the case of your API what is passed in as the "extension" here? If you're loading glTF files then there won't be a magic bytes token and we'll fall back to `extension` which likely won't work for you. Should we set it to this?

```js
const fileType = readMagicBytes( buffer ) || extension || 'gltf';
```

Reply:
hmm, yes. I see the problem here - assuming that everything that doesn't have magic bytes is glTF-json doesn't feel right (and in fact it could actually be a external-tileset json as well).

Maybe we can add an instance-method (something like `TilesRenderer.getContentType(extension: string, buffer: ArrayBuffer): string`),  that can be overridden for a scenario where neither the content-type nor the extension gets a good result.

Reply:
I'm okay assuming that a model is a fallback to gltf for now - I just want to make sure it works for your use case. Is `extension` null or undefined when your extension-less URLs are processed?

Reply:
It's not too relevant for my use case. As far as I know, all of the meshes will be in glTF-binary with embedded textures, so that should be fine either way.